web/src/main/java/se/inera/intyg/rehabstod/web/controller/api/dto/SickLeavesResponseDTO.java

```diff
@@ -20,20 +20,14 @@
 package se.inera.intyg.rehabstod.web.controller.api.dto;
 
 import java.util.List;
+import lombok.Data;
 import se.inera.intyg.rehabstod.web.model.SjukfallEnhet;
 
+@Data
```

Comment:
For serializing to work, we need to add a default constructor. Otherwise we will get an error.